hummingbird/ml/_topology.py

```diff
@@ -242,6 +248,103 @@ def fix_graph(graph):
         else:
             container = PyTorchSklearnContainerClassification
 
-    hb_model = container(hb_model, extra_config)
+    n_threads = None if constants.N_THREADS not in extra_config else extra_config[constants.N_THREADS]
+    batch_size = None if constants.BATCH_SIZE not in extra_config else extra_config[constants.BATCH_SIZE]
+    hb_model = container(hb_model, n_threads, batch_size, extra_config=extra_config)
```

Comment:
If you set `extra_config[constants.N_THREADS]`, then why is it a separate parameter to `container`?  It looks like you pass `n_threads` to `container` redundantly: both explicitly and as part of the `extra`.
(same comment for batch)

Reply:
Good point. I am basically adding the `n_threads` and `batch_size` to the signature of the container (easier to see what is needed for a container to run!?). `extra_config` is needed because some backends need to extra configuration (e.g., tvm).